### PR TITLE
Add support for cardStyle prop

### PR DESCRIPTION
**Pull request by juhasuni**
Currently it is not possible to set the style for an individual card in the CardStack when using the default StackNavigator. My proposal is to add `cardStyle` option to `stackConfig`.

### CODE REVIEW

File: src/navigators/StackNavigator.js [hunk 1]
```diff
@@ -42,6 +43,7 @@ export default (routeConfigMap: NavigationRouteConfigMap, stackConfig: StackNavi
       headerComponent={headerComponent}
       headerMode={headerMode}
       mode={mode}
+      cardStyle={cardStyle}
```
Comment (satya164) [inline src/navigators/StackNavigator.js]: How does it get passed to the card?

**Comment by satya164** [comment]
Is it the same as https://github.com/react-community/react-navigation/issues/8 ?

File: src/navigators/StackNavigator.js (hunk 1 above)
Comment (juhasuni) [inline src/navigators/StackNavigator.js]: CardStack seems to pass it. I didn't have to do anything for it.

**Comment by juhasuni** [comment]
Yes. This resolves #8

**Comment by juhasuni** [comment]
The proposed `screenStyle` is more universal, so if you need to support something like this in other navigators, it might be better to use `screenStyle` instead of `cardStyle`.

**Comment by satya164** [comment]
I don't think we want this in other navigators. Both tab and drawer navigators don't set a background color, and if user is able to override their styles, they'll break.

**Comment by ericvicenti** [comment]
Ok, lets ship this!

**Pull request merged**